.github/workflows/main-build-ubuntu.yml

```diff
@@ -19,6 +19,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v2
+        with:
+          languages: cpp, python

```

Comment:
we don't have building code in python, there's no need to add it to CodeQL

Reply:
what's the current extent of Python usage in the repo? And couldn't we benefit from the static analysis of codeql on those?

Reply:
Jeff replied below btw